packs/data/feat-effects.db/effect-harden-flesh.json

```
@@ -0,0 +1,67 @@
+{
+    "_id": "Np3OSqKxuB9rTbij",
+    "img": "systems/pf2e/icons/spells/flesh-to-stone.webp",
+    "name": "Effect: Harden Flesh",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Harden Flesh]{Harden Flesh}</p>\n<p>You gain resistance 3 to physical damage, except adamantine, until the beginning of your next turn.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 10
+        },
+        "rules": [
+            {
+                "exceptions": [
+                    "adamantine"
+                ],
+                "key": "Resistance",
+                "type": "physical",
+                "value": {
+                    "brackets": [
+                        {
+                            "end": 11,
+                            "start": 10,
+                            "value": 3
+                        },
+                        {
+                            "end": 15,
+                            "start": 12,
+                            "value": 4
+                        },
+                        {
+                            "end": 19,
+                            "start": 16,
+                            "value": 5
+                        },
+                        {
+                            "start": 20,
+                            "value": 6
+                        }
+                    ],
+                    "field": "actor|system.details.level.value"
```

```suggestion
                    ]
```
Brackets default to actor level so we can save a tiny bit of space leaving that off.